### DISCUSSION TITLE
Help Logging In

### DISCUSSION THREAD

**Discussion by TexanScot** [requarks/wiki#5773] [Help / Questions]
I hope someone can help me because I have already built out quite a bit of documentation but now cannot access my wiki!

So for some reason my browser crashed and now Wiki.js will not accept the credentials that I'm pretty sure I setup when I first ran the application. I tried using the 'Forgot my password' option however I get an "The mail configuration is incomplete or invalid." when I enter the email that I used to configure the account.

I'm running wiki.js in a container on Ubuntu 20.04 using the default SQLite datebase. Can anyone give me advice on how to regain access to my wiki.

Thanks in advance.

**Comment by Phil-hostwiki** [comment]
To change your password via the container.

Use https://bcrypt-generator.com/ to generate a bcrypt hash of your new password with 12 rounds.  

1. SSH into your server and run  
2. `docker exec db psql -U wiki -d wiki -c "UPDATE users SET password = 'HASH-PASSWORD' WHERE email = 'YOUR-EMAIL';"`

i.e `docker exec db psql -U wiki -d wiki -c "UPDATE users SET password = '\$2a\$12\$SSDaWdFf.oxxBPLEOxv7iJMFl8JIjYwgxV52gnuYM3Lqdh8i' WHERE email = 'user@example.org';"`

PS. I used backslashes to escape the $ signs in the hashed password.

**Reply by TexanScot** [reply]
Thanks for this reply. I didn't actually use it in the end as I found a post on the Wiki.js support forum that detailed how to manually reset the SQLite password using DB Browser. [Go here ](https://docs.requarks.io/en/troubleshooting) and look for the heading "How to Manually Reset the Admin Password".

I followed the instructions and was successful!
